### ISSUE TITLE
u-*-sizeFit always overridden by percentage size utilities

### ISSUE THREAD

**Issue by tylersticka** [suitcss/utils-size#39]
We noticed on a recent project that the responsive `u-sizeFit` utilities don't seem to work when any `u-size*` utility that sets a percentage width is used.

Example:

```html
<div style="display: flex;">
  <div class="u-sizeFit u-md-size1of2">
    u-sizeFit u-md-size1of2
  </div>
</div>

<div style="display: flex;">
  <div class="u-size1of2 u-md-sizeFit">
    u-size1of2 u-md-sizeFit
  </div>
</div>
```

[See CodePen](https://codepen.io/tylersticka/pen/6f37f41207df036c2566f8217424d5f5?editors=1000)

In this example, the first chunk works as intended. But the second example does not; the child `<div>` will always be `50%` width.

(SUIT CSS is awesome and we love y'all, btw. ❤️)

**Comment by simonsmith** [comment]
Thanks for spotting this @tylersticka 

Looks like we need to set `width: auto` and not just the `flex-basis`. I'll get a fix and some tests added asap.

**Comment by simonsmith** [comment]
Fixed in `2.0.1`. Let me know if that works

**Comment by tylersticka** [comment]
![](http://lol.tylersticka.com/thankyou3.gif)
